docs/subsystems/typing-indicators.md

```diff
@@ -157,24 +170,12 @@ not let the notifications become too "sticky" either.
 
 ## Roadmap
 
-This document is being written just under a year after typing
-indicators were first implemented. The feature has been popular,
-and after some initial cleanup, it has not required a lot of
-maintenance.
-
 The most likely big change to typing indicators is that we will
 add them for stream conversations. This will require some thought
 for large streams, in terms of both usability and performance.
 
 Another area for refinement is to tune the timing values a bit.
-Right now we are probably too aggressive about sending `stop`
-messages, when users often are just pausing. It's possible
+Right now, we are possibly too aggressive about sending `stop`
+messages when users are just pausing to think. It's possible
 to better account for typing speed or other heuristic things
 like how much of the message has already been typed.
-
-From an infrastructure perspective, we should be mindful of
-bandwidth concerns. A fairly easy change would be to send
-user ids instead of emails.
-
-Some folks may want to turn off typing indicators, so we will
-eventually want customized settings here.
```

Comment:
Both of these have been implemented:
- Support for user IDs was implemented in Zulip 2.0.0
- User-level privacy settings were added in Zulip 5.0 (feature level 105)

I created a section above noting the privacy setting.